src/config/my-dspace.interface.ts

```diff
@@ -0,0 +1,19 @@
+import { Config } from './config.interface';
+
+/**
+ * Config that determines some aspects of the MyDSpace section
+ */
+export interface MyDSpaceConfig extends Config {
+  /**
+   * The list of additional metadatas to display at the end of the item component
+   */
+  additionalMetadatas: AdditionalMetaConfig[]
+}
+
+/**
+ * Config that represents an additional metadata.
+ * Contains a value matching a metadata code.
+ */
+export interface AdditionalMetaConfig extends Config {
+  value: string;
```

Comment:
If you just make it a list of strings you can skip the `.map(m => m.value)` (would also make the config more readable)